go/vt/discovery/healthcheck.go

```diff
@@ -187,6 +199,32 @@ func (e *TabletStats) DeepEqual(f *TabletStats) bool {
 			(e.LastError != nil && f.LastError != nil && e.LastError.Error() == f.LastError.Error()))
 }
 
+// GetTabletHostPort formats a tablet host port address
+func (e TabletStats) GetTabletHostPort() string {
+	vtPort := e.Tablet.PortMap["vt"]
+	return netutil.JoinHostPort(e.Tablet.Hostname, vtPort)
+}
+
+// GetHostNameLevel returns the specified hostname level. If the level does not exist it will pick the closest level.
```

Comment:
This is currently unused. I assume you want to use it in your custom templates?

Please document how to use this and why it must be kept in the code.

Also, please add a test for this method. This will better demonstrate the intended use and also make sure that this functionality won't be broken.

Reply:
Ok done.